app/assets/javascripts/application.js

```diff
@@ -7,8 +7,6 @@
 //= require ../../../node_modules/govuk_frontend_toolkit/javascripts/vendor/polyfills/bind.js
 //= require ../../../node_modules/jquery/dist/jquery.js
 //= require ../../../node_modules/hogan.js/web/builds/3.0.2/hogan-3.0.2.js
-//= require ../../../node_modules/d3/d3.js
-//= require ../../../node_modules/c3/c3.js
```

Comment:
There is a `_c3.scss` file we can ditch as well, I think

Reply:
There was also `views/_statistics.scss` which I've now got rid of too.